### ISSUE TITLE
Stop loading uploader schema from lds schema loader

### ISSUE THREAD

**Issue by strk** [linz/linz-lds-bde-schema#192]
The `linz-lds-bde-schema-load` script invokes `linz-bde-uploader-schema-load` but does not pass the `--readonly` switch to it, when specified on commandline.

This ticket is to decide how to behave. The problem is that if `linz-bde-uploader-schema-load` does not support `--readonly`, then passing it over would result in a failure. Is it ok to fail in that case or should we create the schema nonetheless, while not retaining `--readonly` capability for bde-uploader ?

In other words, should passing `--readonly` to the `linz-lds-bde-schema-load` result in a readonly schema of BOTH lds and bde_control schemas ?

NOTE: linz-bde-schema is NOT even called by the script, I can't remember now why the inconsistency of calling one and not the other...

\cc @imincik

**Comment by strk** [comment]
Ok, I made further analisys: `linz-bde-uploader-schema-load` doesn't invoke `linz-bde-schema-load` as well, and `bde_prepare_db` calls each of the required schema loaders anyway, so I now think we should just _stop_ calling `linz-bde-uploader-schema-load` from `linz-lds-bde-schema-load` and expect a failure if such thing is not done in advance, which is what we do if `linz-bde-schema` is not called in advance.

Do you agree @palmerj @imincik ?

**Comment by imincik** [comment]
yes, I agree.
